translations/es/LC_MESSAGES/messages.po

```diff
@@ -428,15 +428,15 @@ msgid ""
 "web. That means if the Google AMP Cache, the URL API or the library fails it "
 "should be possible for websites and consumption apps to gracefully degrade. "
 "If something works with an AMP cache it should also work without a cache."
-msgstr ""
+msgstr "Asegúrate de que, en el caso de que se produzcan interrupciones en AMP o de que aparezca algún problema, no afecta al resto del sitio web. Esto quiere decir que si Google AMP Cache, la API de la URL o la biblioteca fallan, la calidad del servicio debe reducirse de la manera más sutil posible. Si algo funciona con una caché de AMP, también debe funcionar sin ella."
 
 #: /content/learn/who/advertisers.yaml
 msgid ""
 "Even the most memorable creative won’t serve its purpose if the ad is slow "
 "and disruptive for a user. AMP Ads are a new way of building ads, making them"
 " as fast as AMP pages to ensure your ads do what they’re meant to do and "
 "deliver the value you’re looking for."
-msgstr ""
+msgstr "Hasta la creatividad más impactante no servirá de nada si el anuncio es lento y el usuario lo ve con interrupciones. Con los anuncios AMP descubrirás una nueva forma de crear anuncios que, además, serán tan rápidos como las páginas AMP. De este modo, puedes asegurarte de que tus anuncios sirven para el propósito para el que han sido creados y de que ofrecen el valor que buscas."
```

Comment:
I was under that assumption too, but I built the site locally with his changes and it works.  Try it and you'll see.